toktok/reviews.md

```diff
@@ -0,0 +1,106 @@
+code and surrounding elements evolve. This page explains to a new GitHub user
+how to engage in a community review.
+
+# Step 1: Look over the PR list
+
+Take a look at the [pulls](pulls.html) page on this website. It lists all the
+repositories that currently have open pull requests. The first column (marked
+with "#") is the PR number. Clicking on a PR number brings you into the
+Reviewable UI. The branch name links to the GitHub pull request itself.
+
+You can start by reading through the PR titles and see if anything catches
+your interest. You can also just randomly click on some PR numbers and enter
+the review. Everyone is allowed to review any number of PRs they like. You can
+post review comments and engage in discussions. You are welcome, and we will
+be happy to learn about any questions, concerns, or thougths you'd like to
+express.
+
+# Step 2: Log in to Reviewable
+
+-   Open the Reviewable link in the "#" column on the page or on GitHub in the
+    PR description itself.
+-   It will ask you for some permissions. You can safely grant them.
+    Reviewable will not write to your repositories, it only needs those to be
+    able to edit comments and add the Reviewable link.
+-   You should now see a page like the one below.
+-   As you can see, there is a **SHOW UNREVIEWED DIFFS** button. When you
+    click this button, you should be able to see the changes under review.
+
+![Step 2](static/img/reviews/step2.png)
+
+# Step 3
+
+-   Scroll down to see the file changes. You will find something like the
+    screenshot below. Here, we see that someone changed a `4` into a `6` in a
+    file named `DHT.c`. The PR description explains why this was done: "Fix
+    logging condition for IPv6 client timestamp updates".
+-   Now comes your review: your task is now to check whether the change makes
+    sense. In this example, we see that `assoc4` was checked in an IPv6 code
+    block. Now that is fixed. Seems legit. :)
+-   If you don't understand what this change does, ask a question. We will be
+    happy to explain what we're doing. This helps not only you, but also
+    anyone in the future who may be digging through the commit history. It
+    also helps us keep a high standard of documenting changes and maintains
+    transparency towards the community and potential new contributors.
+-   To ask a question or express a concern with a change, click the line you
+    want to comment on. This opens a text editor where you can write your
+    comment in
+    [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+    format.
+
+![Step 3](static/img/reviews/step3.png)
+
+# Step 4
+
+When you are done with a file, click the eye symbol in the corner. This marks
+the file as reviewed. Even if you have put comments on it, still mark it as
+reviewed, which just means that you saw it. It does not mean that you consider
+the file OK. Your comments need to be explicitly marked resolved by you
```

Comment:
If for any reason you're not 100% sure about the file, leave a comment. That way even though you've seen the file, (according to reviewable) everyone knows why, or what you're waiting on.